perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -695,12 +700,15 @@ sub setup_categories_and_bodies : Private {
         push @category_options, $seen{_('Other')} if $seen{_('Other')};
     }
 
+    @body_names_to_list = map { $_->name } values %bodies_to_list;
```

Comment:
This presumably needs to be renamed to list_of_names and dealt with the same way as Around.pm, for when you've visited /report/new directly